_source/user-guide/alerts/configure-alerts.md

```diff
@@ -10,101 +10,166 @@ contributors:
   - imnotashrimp
 ---
 
-To create a new alert, you'll start in Kibana so you can test the query you want to use. After that, you'll continue to the Create a New Alert page, where you can configure the alert and notification settings.
+When you set out to create an alert,
+you'll need to start with a query.
+If you haven't formed your query yet,
+we recommend starting off in Kibana
+so you can see the results of the query in real time.
```

Comment:
Just making sure you have it in a note to change this once we have live preview in the alerts page :)

Reply:
Duly noted :)